crate/get-help.md

```diff
@@ -1 +1,18 @@
-[the public Crate community on Slack](https://crate.io/docs/support/slackin/)
+## Issues
+
+Please report issues via the [GitHub issue tracker](https://github.com/crate/docker-crate/issues).
```

Comment:
I would also link the crate/crate repo
Docker related issues should be reported to crate/docker-crate, database issues to crate/crate